tools/config.py

```diff
@@ -1,15 +1,15 @@
 import os
 import requests
 
-# Update with the MeiliSearch version TAG you want to build the image with
+# Update with the Meilisearch version TAG you want to build the image with
 
 MEILI_CLOUD_SCRIPTS_VERSION_TAG = 'v0.25.2'
 
 # Script settings
 
 DIGITALOCEAN_ACCESS_TOKEN = os.getenv('DIGITALOCEAN_ACCESS_TOKEN')
 DIGITALOCEAN_END_POINT = 'https://api.digitalocean.com/v2'
-SNAPSHOT_NAME = 'MeiliSearch-{}-Debian-10.3'.format(
+SNAPSHOT_NAME = 'Meilisearch-{}-Debian-10.3'.format(
```

Comment:
Maybe this should remain the same? Or do you think this will only impact the next ones?

Reply:
No this is the name of the snapshot we choose to create so normally it won't change anything except the name 😃

Reply:
![Screenshot 2022-01-26 at 21 01 10](https://user-images.githubusercontent.com/16155041/151237963-1cc15225-6392-4fc8-ab6a-db133e096e21.png)